frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -81,6 +81,30 @@ describe("scenarios > dashboard", () => {
     cy.findByText("How many orders were placed in each year?");
   });
 
+  it("should allow empty card title (metabase#12013)", () => {
+    cy.intercept("GET", "/api/table/**/query_metadata").as("tableMetadata");
+    visitDashboard(1);
+
+    // 3 tables
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
+
+    cy.findByTextEnsureVisible("Orders");
+    cy.get("[data-testid='legend-caption']").should("exist");
+
+    cy.findByTextEnsureVisible("Quantity").realHover();
+    cy.icon("pencil").click({ force: true });
+    cy.icon("palette").click();
+
+    cy.get("[data-testid='card.title']")
+      .click()
+      .clear();
+    cy.get("[data-metabase-event='Chart Settings;Done']").click();
```

Comment:
Could this be replaced by `cy.button("Done")`?

Reply:
Something it didn't work, maybe because of the modal?
Perhaps @nemanjaglumac has some idea?

Reply:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/31325167/165862804-b656b256-8d47-4287-ab92-9272fa20bcd0.png">

```js
cy.get(".Modal").within(() => {
  cy.findByDisplayValue("Orders")
    .clear()
    .blur();
  cy.button("Done").click();
});
```

Reply:
Unfortunately I don't think the button is correctly named as `Done` so this won't work yet, so I'll leave it as is for now.